docs/sphinx/reference-properties.rst

```diff
@@ -664,6 +664,8 @@ Property Modification Functions
    :param    val:  The actual string value stored and will be returned by :c:func:`obs_data_get_string`
    :returns: The index of the list item.
 
+   `Note` : If ``p`` is of type OBS_COMBO_TYPE_EDITABLE, :c:func:`obs_data_get_string` will return ``name`` instead of ``val``.
```

Comment:
Does "Note" need backticks? Is it just to provide highlight?

Reply:
It is just for highlight. Here's comparative screenshots. Do tell if you'd rather I remove the backticks.
![Selection_1853](https://github.com/obsproject/obs-studio/assets/37129613/7ca313b1-6f73-4586-9d21-49ffeabd232b)

![Selection_1854](https://github.com/obsproject/obs-studio/assets/37129613/8f45852d-727c-444d-bbc4-a31c6bbddb36)